### ISSUE TITLE
'matches' not triggered if the repeated field is empty

### ISSUE THREAD

**Issue by giorgiopagnoni** [mwfire/titanium-module-validation#4]
leaving the repeated field empty (es. repeatPasswordField) doesn't trigger the rule 'matches'

**Comment by mwfire** [comment]
Thanks, I will check on that!

**Comment by mwfire** [comment]
Sorry, I did not make it yet :( I already have a new version with several improvements on the way, but it will take some time.

One thing that should work is, however, setting the repeatPasswordField to required. I don't know if your input fields are mandatory, but if so, this should do the trick.

**Comment by giorgiopagnoni** [comment]
Yep that's what I am doing right now; maybe I should have mentioned it as a possible temporary solution in my first post. So don't worry, this fix is low priority (for me at least) anyway!
